src/components/atoms/confirmBtn/ConfirmBtn.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { BtnContainer } from "./ConfirmBtn.styles";
+
+const ConfirmBtn = ({ onClick, onModalOpen }) => {
+  const handleClick = () => {
```

Comment:
you don't need to use this function and you definitly don't need to use 2 functions here. Please refactor

Reply:
done